docs/01 - Running & Configuring the Application/A - Running & Configuring the Application.md

```diff
@@ -0,0 +1,154 @@
+# Running & Configuring the Application
+
+- [Running & Configuring the Application](#running--configuring-the-application)
+  - [Running the Application from Source Code](#running-the-application-from-source-code)
+  - [Configuring the Project](#configuring-the-project)
+    - [Config Files](#config-files)
+      - [Path to Yaml](#path-to-yaml)
+        - [Yaml Search](#yaml-search)
+    - [Environment variables](#environment-variables)
+    - [Event Messaging](#event-messaging)
+    - [JVM-Argument based run-configuration](#jvm-argument-based-run-configuration)
+  - [Docker](#docker)
+  - [Running the Application from Docker](#running-the-application-from-docker)
+  - [Incoming Payments Observer](#incoming-payments-observer)
+
+> Created by [gh-md-toc](https://github.com/ekalinin/github-markdown-toc.go)
+
+## Running the Application from Source Code
+
+This section covers how to run the application from source code using the provided Kafka docker image configuration, the Anchor Reference server and the secrets provided in the demo configuration files.
+
+1. Clone this repository: `git clone ssh://git@github.com:stellar/java-stellar-anchor-sdk.git`.
+2. Start a Kafka Queue service: `cd docs/resources/docker-examples/kafka && docker compose up`
+3. Start the Anchor Reference server: `./gradlew service-runner:bootRun --args=--anchor-reference-server`
+    - This would use the default configuration file at [`java-stellar-anchor-sdk/anchor-reference-server/src/main/resources/anchor-reference-server.yaml`], but you can use a custom configuration file by setting the `REFERENCE_SERVER_CONFIG_ENV` environment variable to the path of the configuration file, following the [Path to Yaml](#path-to-yaml) format.
```

Comment:
Maybe just link to the file instead of providing the whole path in text? This goes for all the paths mentioned.